scripts/test/modules/bins.py

```diff
@@ -0,0 +1,234 @@
+import os
+import threading
+import subprocess
+import fcntl
+
+import time
+
+from logger import log
+
+PLUGIN_DIR = "/etc/snap/plugins"
+SNAP_DIR = "/usr/local/bin"
+
+PLUGIN_URL = "http://snap.ci.snap-telemetry.io/plugin/build/latest/snap-plugin-collector-psutil"
+SNAP_URL = "http://snap.ci.snap-telemetry.io/snap/master/latest/snapd"
+SNAPCTL_URL = "http://snap.ci.snap-telemetry.io/snap/master/latest/snapctl"
+
+
+def _non_block_read(output):
```

Comment:
This test is being executed in docker container, I don't thing we have to worry about it